ocs_ci/framework/conf/default_config.yaml

```diff
@@ -70,6 +70,8 @@ DEPLOYMENT:
   preserve_bootstrap_node: False
   # Following options are defaults for the max Noobaa endpoints
   noobaa_endpoints: 2
+  # Terraform version
+  terraform_version: "0.12.26"
```

Comment:
should we put this to OCP 4.5 related config as well?  https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.5-config.yaml

Thiking about to have this defined as `default_terraform_version`

Reply:
Done